src/pages/profile.tsx

```diff
@@ -1 +1,2 @@
 export { Profile as default } from "profile/Profile";
+export { redirect as getServerSideProps } from "shared";
```

Comment:
I think I'd prefer client side auth lookup instead of SSR for most of these as we then get a quicker Time to First Byte and the website feels more responsive without the 'large' load times between pages as we check user auth.